Sources/Shared/ViewModel.swift

```diff
@@ -36,6 +37,8 @@ public struct ViewModel: Mappable {
 
   /// The index of the ViewModel when appearing in a list, should be computed and continuously updated by the data source
   public var index = 0
+  /// An optional identifier for your data
+  public var identifier: Int?
```

Comment:
Fair point, I have this weird Obj-C association with `id`. Also whenever I use `id`, semantically I just assume that it is non-optional. Where as `identifier` to me feels like it could be up to the implementation. Even if `id` is short for `identifier`... my mind works in mysterious ways sometimes 😎 

But we can totally go for `id` if we think that is better.